pkg/config/git.go

```diff
@@ -53,15 +56,19 @@ func (r *GitRepository) setDefaults() error {
 
 			remote.Name = "origin"
 		}
+		if remote.Interval == 0 {
```

Comment:
```suggestion
		if remote.Interval < 1 {
```

suggestion: since duration can be negative